packages/hardhat/hardhat.config.ts

```diff
@@ -1,10 +1,14 @@
 import * as dotenv from "dotenv";
 dotenv.config();
 import { HardhatUserConfig } from "hardhat/config";
-import "@nomicfoundation/hardhat-toolbox";
+import "@nomicfoundation/hardhat-ethers";
+import "@nomicfoundation/hardhat-chai-matchers";
+import "@typechain/hardhat";
+import "hardhat-gas-reporter";
+import "solidity-coverage";
+import "@nomicfoundation/hardhat-verify";
 import "hardhat-deploy";
-import "@matterlabs/hardhat-zksync-solc";
-import "@matterlabs/hardhat-zksync-verify";
+import "hardhat-deploy-ethers";
```

Comment:
1. Removed`@nomicfoundation/hardhat-toolbox` since  [`hardhat-deploy-ethers`](https://github.com/wighawag/hardhat-deploy-ethers) now tells you to add things manually in-order to make things properly work inline with `harhdat-deploy` nd `hardhat-deploy-ethers`

2. Removed `ZkSync` stuff, since it was not working properly already and checkout out their docs, it seems there are some zkSync specific stuff which should be added to other networks like `{zksync: false}`  checkout this https://era.zksync.io/docs/tools/hardhat/migrating-to-zksync.html#configuration-changes. I think maybe we can let people configure it